src/utils/resources.py

```diff
@@ -1106,13 +1105,25 @@ def __init__(self, properties: Dict[str, Any], *args, **kwargs):
             if isinstance(values.get("packages"), str):
                 values["packages"] = [value.strip() for value in values["packages"].split(",")]  # type: ignore
 
+            if isinstance(values.get("packages_from_raw_bash"), str):
+                out, err = self.check_output_bash_snippet(values.get("packages_from_raw_bash"))
+                if err:
+                    logger.error(
+                        "Error while running apt resource packages_from_raw_bash snippet for '" + str(key) + "' extras:"
+                    )
+                    logger.error(err)
+                values["packages"] = [value.strip() for value in out.split("\n")]
```

Comment:
```suggestion
                values["packages"] = values.get("packages", []) + [value.strip() for value in out.split("\n")]
```

(to append + cover the case where `packages` was not provided at all)